imports/plugins/included/marketplace/client/templates/stripeConnectSignupButton/stripeConnectSignupButton.js

```diff
@@ -47,11 +48,15 @@ Template.stripeConnectSignupButton.events({
     const businessName = shop.addressBook[0].company;
     const streetAddress = shop.addressBook[0].address1;
     const city = shop.addressBook[0].city;
-    const state = shop.addressBook[0].state;
+    const state = shop.addressBook[0].region;
     const zip = shop.addressBook[0].postal;
 
+    const user = Meteor.user() || {};
+    const { address: email } = _.first(user.emails) || {};
+    const [firstName, lastName] = _.split(user.name, " "); // split on " " isn't perfect, but we use it elsewhere, so it's consistent. to that point, consider adding a helper to User
```

Comment:
a couple issues here:
1. I don't think we need to use lodash here, as [string.prototype.split](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split) should work fine here.
2. The bigger issue here is that I think there are some edge cases where this will leave off parts of someone's name. E.g. my name is `Steven Spencer Norman Jr.` and if I have all of that here, this will split first name and last name into `Steven Spencer` and not `Steven Norman` - I think the easy solution here would be to use spread into the rest of the array.. e.g.
```
const [firstName, ...last] = user.name.split(" ");
const lastName = last.join(" ");
```